repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
```

Comment:
The use case is that I need a way to debug without ssh-ing into the machines, and there's no way to run `kubectl exec` or `kubectl logs` currently through the admin-router.

If we want to revert this later, once that becomes possible, then I don't have a problem with that. But I don't think hypothetical future security for a product with no current security is a good reason to keep usability miserable.